src/main/java/core/selenium/webdrivers/FireFox.java

```diff
@@ -0,0 +1,25 @@
+package core.selenium.webdrivers;
+
+import io.github.bonigarcia.wdm.FirefoxDriverManager;
+import org.openqa.selenium.WebDriver;
+
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.firefox.FirefoxOptions;
+
+/**
+ * This class implements the configuration of the Firefox browser.
+ *
+ * @author Josue Rodriguez Garcia.
+ * @version 0.0.1
+ */
+public class FireFox {
+    /**
+     * This method is used for configure the Firefox browser.
+     * @return a WebDriver with configuration the Firefox browser.
```

Comment:
Please add a line to separate the description of the return.

Reply:
Solved